CONTRIBUTING.md

```diff
@@ -177,8 +177,8 @@ Exceptions:
 | `enhancement` | [search][search-label-enhancement] | Feature requests. |
 | `bug` | [search][search-label-bug] | Confirmed bugs or reports that are very likely to be bugs. |
 | `help wanted` | [search][search-label-help-wanted] | The core team would appreciate help from the community in resolving these issues. |
-| `beginner` | [search][search-label-beginner] | Less complex issues which would be good first issues to work on for users who want to contribute. |
-| `info needed` | [search][search-label-more-information-needed] | More information needs to be collected about these problems or feature requests (e.g. steps to reproduce). |
+| `good first issue` | [search][search-label-good-first-issue] | Less complex issues which would be good first issues to work on for users who want to contribute. |
+| `info needed` | [search][search-label-info-needed] | More information needs to be collected about these problems or feature requests (e.g. steps to reproduce). |
```

Comment:
Did you mean to include this change as well?

Reply:
Yes. thx.  bug fix along the way.  I'll update the description.